members/Ko.md

```diff
@@ -0,0 +1,13 @@
+# Ko Szu Yuan
+
+## Poly Diploma
+
+## No Work Experience
+
+* [3 months: 2017 May - July] Sofware Engineering Intern, at PayPal Singapore
```

Comment:
You said that you have no work experience but talk about Software Engineering Intern here??